src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -214,6 +214,7 @@ class ZWaveJSNodeConfig extends LitElement {
             <zwave_js-custom-param
               .hass=${this.hass}
               .deviceId=${this.deviceId}
+              .onNewValue=${this._fetchData}
```

Comment:
The context of this function will be wrong, also `fetchData` doesn't meet the typing for `onNewValue` set in `zwave_js-custom-param`